src/views/v2/components/events-bar/search-button/icon.php

```diff
@@ -10,7 +10,6 @@
  * @link http://m.tri.be/1aiy
  *
  * @version 4.9.10
```

Comment:
every template that has an update should reset the version to TBD

```suggestion
 * @version TBD
```